buildkite/common

```diff
@@ -32,4 +32,5 @@ export APP_DOCKER_COMMIT_TAG=$ORG/hint:$GIT_ID
 export APP_DOCKER_BRANCH_TAG=$ORG/hint:$GIT_BRANCH
 export USER_CLI_COMMIT_TAG=$ORG/hint-user-cli:$GIT_ID
 export USER_CLI_BRANCH_TAG=$ORG/hint-user-cli:$GIT_BRANCH
-
+export USER_CLI_COMMIT_TAG_NEW=$ORG/hint-user-cli:$GIT_ID
+export USER_CLI_COMMIT_TAG_OLD=$ORG-hint-user-cli:$GIT_ID
```

Comment:
I'm confused by this and the above comment in `build-cli.sh` - what is going on here?